hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/UniqueTokenValue.java

```diff
@@ -134,6 +134,20 @@ public VirtualValue asReadOnly() {
         return copy;
     }
 
+    // Keep it in sync with serializeTo()
+    int getSerializedSize() {
+        return Long.BYTES // owner account num
+                + Long.BYTES // spender account num
+                + Long.BYTES // packed creation time
+                + 1
+                + Math.min(metadata.length, MAX_METADATA_BYTES) // metadata len + metadata
+                + Long.BYTES
+                + Long.BYTES // prev NFT num pair: token num + serial num
+                + Long.BYTES
+                + Long.BYTES; // next NFT num pair: token num + serial num
+    }
+
+    // Keep it in sync with getSerializedSize()
```

Comment:
Most of these have comments stating what each number of bytes represents, but some don't. Helpful to add for the lines that don't have comments?